### PR TITLE
Fix: user saved plan location format

### PR DESCRIPTION
**Pull request by timwangmusic**
## Description
When user auto completes the location with search input, the resulted string in search result page URL does not follow traditional formatting for city, country names, etc.

## Solution
Fix the format while posting the plan to be saved to the backend.

### CODE REVIEW

File: assets/js/search_results.js [hunk 1]
```diff
@@ -78,6 +81,19 @@ function planToView(plan, planIndex) {
     return view;
 }
 
+// The autocompleted locations need to be fixed. City and country names need to be capitalized and admin level 2 names need to be changed to all upper cases.
+function correctLocationName(location) {
```
Comment (rwangsc18) [inline assets/js/search_results.js:85]: nit: call it normalizeLocation, since we just align the format to make things consistent

**Review by rwangsc18 — COMMENTED** [review]
LGTM

File: assets/js/search_results.js (hunk 1 above)
Comment (timwangmusic) [reply assets/js/search_results.js]: Done

**Review by rwangsc18 — APPROVED** [review]
LGTM

**Pull request merged**